trunk/SUAVE/Input_Output/VTK/save_vehicle_vtk.py

```diff
@@ -0,0 +1,152 @@
+## @ingroup Input_Output-VTK
+# save_vehicle_vtks.py
+# 
+# Created:    Jun 2021, R. Erhard
+# Modified: 
+#  
+
+from SUAVE.Core import Data
+from SUAVE.Methods.Aerodynamics.Common.Fidelity_Zero.Lift.generate_wing_vortex_distribution  import generate_wing_vortex_distribution
+from SUAVE.Input_Output.VTK.save_wing_vtk import save_wing_vtk
+from SUAVE.Input_Output.VTK.save_prop_vtk import save_prop_vtk
+from SUAVE.Input_Output.VTK.save_prop_wake_vtk import save_prop_wake_vtk
+from SUAVE.Input_Output.VTK.save_fuselage_vtk import save_fuselage_vtk
+
+
+def save_vehicle_vtks(vehicle, settings, Results, time_step, prop_filename="propeller.vtk", rot_filename="rotor.vtk",
+                     wake_filename="prop_wake.vtk", wing_filename="wing_vlm.vtk", fuselage_filename="fuselage.vtk", save_loc=None):
+    """
+    Saves SUAVE vehicle components as VTK files in legacy format.
+
+    Inputs:
+       vehicle                Data structure of SUAVE vehicle                    [Unitless] 
+       settings               Settings for aerodynamic analysis                  [Unitless] 
+       Results                Data structure of wing and propeller results       [Unitless]
+       time_step              Simulation time step                               [Unitless]
+       prop_filename          Name of vtk file to save                           [String] 
+       rot_filename           Name of vtk file to save                           [String] 
+       wake_filename          Name of vtk file to save                           [String] 
+       wing_filename          Name of vtk file to save                           [String] 
+       fuselage_filename      Name of vtk file to save                           [String] 
+       save_loc               Location at which to save vtk files                [String]
+       
+    Outputs:                                   
+       N/A
+
+    Properties Used:
+       N/A 
+    
+    Assumptions:
+       Quad cell structures for mesh
+    
+    Source:  
+       None
+    
+    """  
+    # unpack vortex distribution 
+    try:
+        VD = vehicle.vortex_distribution 
+    except:
+        settings = Data()
```

Comment:
Wait, how would you have results if you didn't already run the VLM. Also wouldn't these points not necessarily match the number of points in the VLM?

I would throw an error instead.

Reply:
My thought was if the vehicle hasn't yet been analyzed, this would allow the user to still export the geometries just without the simulation data. I realize now to do this I needed to add additional checks when writing the results to each file. I added a check in the required scripts to see if there is data to be exported for the cells/nodes. If no data, it just exports the geometries of the components.